setup.py

```diff
@@ -105,6 +105,7 @@ def run(self):
             check_call('selftests/lint.sh')
             check_call('selftests/signedoff-check.sh')
             check_call('selftests/spell.sh')
+            check_call('selftests/modules-boundaries.sh')
```

Comment:
Hi @richtja, thanks for this. Maybe since you are adding this, we could take this opportunity to remove the need for relative imports on core, as we discussed here: #4492  and #3525 . What you think?

Reply:
Right, I will update the script.